artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/connect/mirror/AMQPMirrorControllerTarget.java

```diff
@@ -0,0 +1,367 @@
+                      server.getIdentity() + "::Received " + message + "\n" +
+                      "*******************************************************************************************************************************");
+      }
+      try {
+         Object eventType = annotationsMap.get(EVENT_TYPE);
+         if (eventType != null) {
+            // I'm not using fancy switch with strings for JDK compatibility, just in case
+            if (eventType.equals(ADDRESS_SCAN_START)) {
+               logger.debug("Starting scan for removed queues");
+               startAddressScan();
+            } else if (eventType.equals(ADDRESS_SCAN_END)) {
+               logger.debug("Ending scan for removed queues");
+               endAddressScan();
+            } else if (eventType.equals(ADD_ADDRESS)) {
+               AddressInfo addressInfo = parseAddress(message);
+               if (logger.isDebugEnabled()) {
+                  logger.debug("Adding Address " + addressInfo);
+               }
+               addAddress(addressInfo);
+            } else if (eventType.equals(DELETE_ADDRESS)) {
+               AddressInfo addressInfo = parseAddress(message);
+               if (logger.isDebugEnabled()) {
+                  logger.debug("Removing Address " + addressInfo);
+               }
+               deleteAddress(addressInfo);
+            } else if (eventType.equals(CREATE_QUEUE)) {
+               QueueConfiguration queueConfiguration = parseQueue(message);
+               if (logger.isDebugEnabled()) {
+                  logger.debug("Creating queue " + queueConfiguration);
+               }
+               createQueue(queueConfiguration);
+            } else if (eventType.equals(DELETE_QUEUE)) {
+               String address = (String) annotationsMap.get(ADDRESS);
+               String queueName = (String) annotationsMap.get(QUEUE);
+               if (logger.isDebugEnabled()) {
+                  logger.debug("Deleting queue " + queueName + " on address " + address);
+               }
+               deleteQueue(SimpleString.toSimpleString(address), SimpleString.toSimpleString(queueName));
+            } else if (eventType.equals(POST_ACK)) {
+               String address = (String) annotationsMap.get(ADDRESS);
+               String queueName = (String) annotationsMap.get(QUEUE);
+               AmqpValue value = (AmqpValue) message.getBody();
+               Long messageID = (Long) value.getValue();
+               if (logger.isDebugEnabled()) {
+                  logger.debug("Post ack address=" + address + " queueName = " + queueName + " messageID=" + messageID);
+               }
+               postAcknowledge(address, queueName, messageID);
+            }
+         } else {
+            if (logger.isDebugEnabled()) {
+               logger.debug("Sending message " + message);
+            }
+            sendMessage(message);
+         }
+      } catch (Throwable e) {
+         logger.warn(e.getMessage(), e);
```

Comment:
This seems questionable given the finally goes on to accept, no matter what.

Reply:
I'm removing it... I had some other call in there during my refactoring.. it's not needed any more..
thanks